### PR TITLE
make.bat: fix path backslashes to forward slashes for more compatibility, change quoting to be more consistent

### PR DESCRIPTION
**Pull request by JalonSolov**
Tested in both cmd and PowerShell with https://github.com/andy-5/wslgit, and in cmd and PowerShell (by @felipensp) with "Git for Windows".

### CODE REVIEW

File: make.bat [hunk 1]
```diff
@@ -42,7 +42,7 @@ if ["%~1"] == [""] goto :init
 REM Target options
 if !shift_counter! LSS 1 (
     if "%~1" == "help" (
-        if not ["%~2"] == [""] set "subcmd=%~2"& shift& set /a shift_counter+=1
+        if not ["%~2"] == [""] set subcmd=%~2& shift& set /a shift_counter+=1
```
Comment (spytheman) [inline make.bat:45]:
```suggestion
        if not ["%~2"] == [""] set subcmd="%~2"& shift& set /a shift_counter+=1
```

File: make.bat [hunk 2]
```diff
@@ -11,27 +11,27 @@ set compiler=
...
+set vc_dir=%~dp0vc
 
 REM Let a particular environment specify their own TCC and VC repos (to help mirrors)
-if /I not ["%TCC_GIT%"] == [""] set "tcc_url=%TCC_GIT%"
-if /I not ["%TCC_BRANCH%"] == [""] set "tcc_branch=%TCC_BRANCH%"
+if /I not ["%TCC_GIT%"] == [""] set tcc_url=%TCC_GIT%
+if /I not ["%TCC_BRANCH%"] == [""] set tcc_branch=%TCC_BRANCH%
 
-if /I not ["%VC_GIT%"] == [""] set "vc_url=%VC_GIT%"
+if /I not ["%VC_GIT%"] == [""] set vc_url=%VC_GIT%
```
Comment (spytheman) [inline make.bat:34]:
```suggestion
if /I not ["%VC_GIT%"] == [""] set vc_url="%VC_GIT%"
```

File: make.bat [hunk 3]
```diff
@@ -11,27 +11,27 @@ set compiler=
...
-set "vc_url=https://github.com/vlang/vc"
-set "vc_dir=%~dp0vc"
+set vc_url=https://github.com/vlang/vc
+set vc_dir=%~dp0vc
 
 REM Let a particular environment specify their own TCC and VC repos (to help mirrors)
-if /I not ["%TCC_GIT%"] == [""] set "tcc_url=%TCC_GIT%"
-if /I not ["%TCC_BRANCH%"] == [""] set "tcc_branch=%TCC_BRANCH%"
+if /I not ["%TCC_GIT%"] == [""] set tcc_url=%TCC_GIT%
+if /I not ["%TCC_BRANCH%"] == [""] set tcc_branch=%TCC_BRANCH%
```
Comment (spytheman) [inline make.bat:32]:
```suggestion
if /I not ["%TCC_BRANCH%"] == [""] set tcc_branch="%TCC_BRANCH%"
```

File: make.bat [hunk 4]
```diff
@@ -11,27 +11,27 @@ set compiler=
...
 REM VC settings
-set "vc_url=https://github.com/vlang/vc"
-set "vc_dir=%~dp0vc"
+set vc_url=https://github.com/vlang/vc
+set vc_dir=%~dp0vc
 
 REM Let a particular environment specify their own TCC and VC repos (to help mirrors)
-if /I not ["%TCC_GIT%"] == [""] set "tcc_url=%TCC_GIT%"
-if /I not ["%TCC_BRANCH%"] == [""] set "tcc_branch=%TCC_BRANCH%"
+if /I not ["%TCC_GIT%"] == [""] set tcc_url=%TCC_GIT%
```
Comment (spytheman) [inline make.bat:31]:
```suggestion
if /I not ["%TCC_GIT%"] == [""] set tcc_url="%TCC_GIT%"
```

File: make.bat [hunk 5]
```diff
@@ -42,7 +42,7 @@ if ["%~1"] == [""] goto :init
 REM Target options
 if !shift_counter! LSS 1 (
     if "%~1" == "help" (
-        if not ["%~2"] == [""] set "subcmd=%~2"& shift& set /a shift_counter+=1
+        if not ["%~2"] == [""] set subcmd=%~2& shift& set /a shift_counter+=1
     )
     for %%z in (build clean cleanall check help rebuild) do (
         if "%~1" == "%%z" set target=%1& shift& set /a shift_counter+=1& goto :verifyopt
```
Comment (spytheman) [inline make.bat:48]:
```suggestion
        if "%~1" == "%%z" set target="%1"& shift& set /a shift_counter+=1& goto :verifyopt
```

File: make.bat [hunk 6]
```diff
@@ -118,7 +118,7 @@ if !flag_local! NEQ 1 (
     pushd "%vc_dir%" && (
         echo Updating vc...
         echo  ^> Sync with remote !vc_url!
-        cd "%vc_dir%"
+        cd %vc_dir%
```
Comment (spytheman) [inline make.bat:121]:
```suggestion
        cd "%vc_dir%"
```

File: make.bat [hunk 7]
```diff
@@ -195,26 +195,26 @@ if "%PROCESSOR_ARCHITECTURE%" == "x86" (
...
+	call "%InstallDir%/Common7/Tools/vsdevcmd.bat" -arch=%HostArch% -host_arch=%HostArch% -no_logo
+) else if exist "%VsWhereDir%/Microsoft Visual Studio 14.0/Common7/Tools/vsdevcmd.bat" (
+	call "%VsWhereDir%/Microsoft Visual Studio 14.0/Common7/Tools/vsdevcmd.bat" -arch=%HostArch% -host_arch=%HostArch% -no_logo
 )
 
 set ObjFile=.v.c.obj
 
 echo  ^> Attempting to build "%V_BOOTSTRAP%" (from v_win.c) with MSVC
-cl.exe /volatile:ms /Fo%ObjFile% /W0 /MD /D_VBOOTSTRAP vc\v_win.c user32.lib kernel32.lib advapi32.lib shell32.lib /link /nologo /out:"%V_BOOTSTRAP%" /incremental:no
+cl.exe /volatile:ms /Fo%ObjFile% /W0 /MD /D_VBOOTSTRAP vc/v_win.c user32.lib kernel32.lib advapi32.lib shell32.lib /link /nologo /out:"%V_BOOTSTRAP%" /incremental:no
```
Comment (spytheman) [inline make.bat:217]: I am not sure that works, is it tested?

cl.exe seems to be using / as an option delimiter too.

Comment (spytheman) [reply make.bat:217]: perhaps `"vc/v_win.c"` ?

Comment (JalonSolov) [reply make.bat:217]: Oops.  Was a bit overzealous there.

**Pull request merged**